app/views/layouts/application.html.haml

```diff
@@ -14,7 +14,7 @@
     %meta{:name => "author", :content => "Diaspora, Inc."}/
 
     %link{:rel => 'shortcut icon', :href => "#{image_path('favicon.png')}" }
-    %link{:rel => 'apple-touch-icon', :href => "#{image_path('apple-touch-icon.png')}"}
+    %link{:rel => 'apple-touch-icon', :href => "#{image_path('favicon.png')}"}
```

Comment:
| Maybe we can merge this migration and then start improving and polish the
design on the top of it, with small pull requests?

That's the way to go!

This is great, a lot of incremental improvements will be possible on top of
BS3. I've been too busy at work to help, I hope we can go on improving the
UX step by step once the framework is solid.



On Thu, May 28, 2015 at 9:00 PM, Benjamin Neff <notifications@github.com>
wrote:

> In app/views/layouts/application.html.haml
> <https://github.com/diaspora/diaspora/pull/6002#discussion_r31289377>:
>
> > @@ -14,7 +14,7 @@
> >      %meta{:name => "author", :content => "Diaspora, Inc."}/
> >
> >      %link{:rel => 'shortcut icon', :href => "#{image_path('favicon.png')}" }
> > -    %link{:rel => 'apple-touch-icon', :href => "#{image_path('apple-touch-icon.png')}"}
> > +    %link{:rel => 'apple-touch-icon', :href => "#{image_path('favicon.png')}"}
>
>  In any case, I suspect there are lots of images that aren't used anymore
> in the interface and that could be suppressed.
> @AugierLe42e <https://github.com/AugierLe42e>: i started cleanup the
> unused images yesterday:
> https://github.com/SuperTux88/diaspora/commits/cleanup-assets ... so you
> don't have to cleanup any images in this PR
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diaspora/diaspora/pull/6002/files#r31289377>.
>